test/integration/freetext_editor_spec.mjs

```diff
@@ -3688,4 +3688,131 @@ describe("FreeText Editor", () => {
       );
     });
   });
+
+  describe("Undo annotation popup has the expected behaviour", () => {
```

Comment:
*Similar comments apply elsewhere throughout the patch as well.*

```suggestion
  describe("Undo editor popup has the expected behaviour", () => {
```

Reply:
I've changed this to `Undo deletion popup has the expected behaviour` across all files, since that seems more relevant, let me know if we can phrase it better.